src/shared/lib/onMobxPromise.ts

```diff
@@ -1,15 +1,32 @@
 import {MobxPromise} from "mobxpromise";
 import {autorun, IReactionDisposer} from "mobx";
 
-export default function onMobxPromise<T>(promise:MobxPromise<T>,
-                                         onComplete:(result:T)=>void,
+export function onMobxPromise<T>(promise:MobxPromise<T>,
+                          onComplete:(result:T)=>void,
```

Comment:
can onComplete fire multiple times as observed MobxPromise go from pending to complet to pending to complete?

Reply:
thats what the `times` parameter determines